client/src/header.css

```diff
@@ -0,0 +1,29 @@
+#head{
```

Comment:
just a suggestion, you should use classes instead of ids for styling elements, the reason is a more preferred choice is we can reuse classes on html elements.

Reply:
yes sir i would have but components have different css properties that's why i went for id